nuxeo-routing-core/src/main/java/org/nuxeo/ecm/platform/routing/core/io/TaskWriter.java

```diff
@@ -259,7 +259,7 @@ protected String getUnprefixedActorId(String actorId) {
 
     protected void writeWorkflowInitiator(JsonGenerator jg, String workflowInitiator) throws IOException {
         if (ctx.getFetched(ENTITY_TYPE).contains(FETCH_WORKFLOW_INITATIOR)) {
-            NuxeoPrincipal principal = userManager.getPrincipal(workflowInitiator);
+            Object principal = fetchActor(workflowInitiator);
```

Comment:
Are you sure this is needed? A priori the `workflowInitiator` shouldn't be prefixed and cannot be a groupname.

Reply:
Just for coherency purposes and code reuse.